examples/humioaggregatealert.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: core.humio.com/v1alpha1
+kind: HumioAggregateAlert
+metadata:
+    name: example-aggregate-alert-managed
+spec:
+  managedClusterName: example-humiocluster
+  name: example-aggregate-alert
+  queryString: "#repo = humio | error = true "
```

Comment:
Both examples should have an aggregate query as examples, so perhaps append ` | count()` to the query strings.